packages/react-reconciler/src/ReactFiberWorkLoop.new.js

```diff
@@ -1674,6 +1671,16 @@ function performUnitOfWork(unitOfWork: Fiber): void {
     next = beginWork(current, unitOfWork, subtreeRenderLanes);
   }
 
+  // TODO (effects) In theory, we need to reset childLanes after beginWork.
+  // If we reset during clone, we lose scheduled updates.
+  // If we copy during clone, we have infinite loops.
+  // The problem seems to be that this reset is too broad;
+  // we still lose certain kinds of updates (e.g. ones scheduled during render with overlapping lanes).
+  //
+  // This would also be more appropriately done within beginWork,
+  // but there are so many return paths there that I put it here for now to try the idea.
+  unitOfWork.childLanes = NoLanes;
```

Comment:
Note to self: I think one of the problems with this approach is that it erases the `OffscreenLane` that might get added by `updateOffscreenComponent` during `beginWork`.